test/DartTest.ts

```diff
@@ -275,6 +277,10 @@ describe('transpile to dart', () => {
       expectTranslate("`\\$a`").to.equal(" '''\\$a''' ;");
     });
 
+    it('escapes escape sequences', () => {
+      expectTranslate("`\\\\u1234`").to.equal(" '''\\\\u1234''' ;");
```

Comment:
No, we're all good. This has four backslashes, because it's TypeScript source that is escaped within a TypeScript string, doubling the number of backslashes. The actual string after parsing has two backslashes, like the Angular source. And that then generates a two-backslash result, as expected and tested for.

Yeah, this is all a bit confusing.